### ISSUE TITLE
Mask2former training configs

### ISSUE THREAD

**Issue by levan92** [open-mmlab/mmdetection#8125]
Hi @PeterVennerstrom @chhluo, thank you so much for your efforts to implement mask2former in mmdetection! 

I have queries regarding the training config parameters for this:

Does the current configs of initial LR=0.0001 assume training on 8x GPU (2 samples per gpu, so effective BS of 16)? If I were to train on a cluster with only 4 GPUs and to get similar results, I should be doubling `max_iters` and corresponding iters parameters, and also the initial LR according to the [linear scaling rule](https://arxiv.org/abs/1706.02677). Just want to confirm my understanding is correct. 

Also, I notice the names of the config files have "*_8x2_*" in them, what is the significance of these values? Does that actually mean 8 gpus from 2 nodes, aka 16 gpus in all? that will be an effective BS of 32 assuming 2 samples per gpu?

**Comment by PeterVennerstrom** [comment]
The 8x2 or 16x1 notation means gpu # x images per gpu.  In both cases the total batch size is 16.  I think 16x1 is used for the Swin-L backbone due to memory requirements.  

Your understanding of the adjustments to max_iters and scaling the learning rate is correct.  In addition see the step parameter in the lr config and the interval for evaluation.

**Comment by levan92** [comment]
Right, got it! Thank you for your timely reply!
